cli/run.go

```diff
@@ -21,14 +21,15 @@ import (
 
 func init() {
 	cmd := register("run", runRun, `
-usage: flynn run [-d] [-r <release>] [-e <entrypoint>] [--] <command> [<argument>...]
+usage: flynn run [-d] [-r <release>] [-e <entrypoint>] [-l] [--] <command> [<argument>...]
 
 Run a job.
 
 Options:
-	-d, --detached   run job without connecting io streams
-	-r <release>     id of release to run (defaults to current app release)
-	-e <entrypoint>  overwrite the default entrypoint of the release's image
+	-d, --detached    run job without connecting io streams (implies --enable-log)
+	-r <release>      id of release to run (defaults to current app release)
+	-e <entrypoint>   overwrite the default entrypoint of the release's image
+	-l, --enable-log  send output to log streams
```

Comment:
I added it in case I want to run a command in the terminal but also have the output go to the logs, e.g. `flynn run --enable-log rake db:migrate`